### ISSUE TITLE
Support for severity levels?

### ISSUE THREAD

**Issue by lebinh** [cloudflare/unsee#151]
Some other monitoring systems have a concept of severity level for alerts, e.g. a high_memory_usage alert is much less important than a service_is_down alert. A way to differentiate (color, shape, size, etc.) in unsee to support severity level would be very helpful in case you have a lot of alert noise and wanted to be sure you won't miss an important alert somewhere. Or some ordering based on severity could help if you want to optimize your works on eliminating them.

A simple implementation would be allow user to config one label to be used as severity level and adding UI support for ordering based on that label (or even better, order based on any label).

**Comment by prymitive** [comment]
There is alert status in prometheus, possible values are `unprocessed`, 'active' and `suppressed`, unsee uses that to color alerts, I'm not very keep on adding more colors or different rendering modes based on some additional alert sub-status (e.g. a warning would be "active, but less active than others").
Sorting is an option, but that depends on how you group alerts, to get any proper sorting your grouping would need to include `severity` label (or whatever someone chooses to use), so that severity is a property of a group and we can sort groups.
But then https://masonry.desandro.com (which is used for alert grid rendering) would need to support, which doesn't seem the case.
It's doable but a bit vague how should that work. You can always color any label and/or filter based on that, so if there's a lot alerts and you want only those with `severity=critical` than you can get those in the UI.

**Comment by lebinh** [comment]
I agree that we probably don't need more color in unsee, it's the most colorful on my desktop already (although I feel the different label colors aren't very useful). I guess a `severity` label would be enough to differentiate to some extend.

However, filtering is different from sorting though. Constantly changing the filter is more work compare to just set a default ordering and be comfortable that I won't miss an important alert, especially when there are a lot of others. Also that'd save sometime when looking for an alert to work on, just look at the top (instead of going up and down looking for what is the next important thing to fix).

Even if the grid framework doesn't support we can use some hack like remove + sort + append again? Assume the performance won't be too terrible and we don't have to do that too often.
